src/IO/ReadBufferFromRemoteFS.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Disks/IDiskRemote.h>
+#include <IO/ReadBufferFromFile.h>
+
+
+namespace DB
+{
+
+class ReadBufferFromRemoteFS : public ReadBufferFromFileBase
+{
+friend class ReadIndirectBufferFromRemoteFS;
+friend class AsynchronousReadIndirectBufferFromRemoteFS;
```

Comment:
I haven't found where it's used

Reply:
It is needed to be able to do `impl->absolute_position` from *InderectReadBuffer's.